src/main/kotlin/org/malachite/estella/mails/MailTexts.kt

```diff
@@ -67,11 +68,12 @@ object MailTexts {
             After first login change password and setup your name. We hope that our site will help you with work.
         """.trimIndent()
 
-    fun getTaskAssignmentRequestText(stage: StageType, url: String, hrPartnerFullName: String, position: String): String =
+    fun getTaskAssignmentRequestText(stage: StageType, url: String, hrPartnerFullName: String, position: String, password: UUID?): String =
```

Comment:
Wydaje mi się że nie powinniśmy pozwalać na nullowalność passworda, bo byłoby kiepsko jakbyśmy wysłali maila z tekstem twoje hasło to null XD. Więc ogólnie checkowałbym i rzucał wyjątek jakby było nullem

Reply:
Ogólnie rzecz biorąc, taka sytuacja nie powinna mieć miejsca, z racji tego, że przypisujemy devów do zadania na podstawie pobranego z bazy zadania. Mogę dodać !!, wtedy rzuci nam internal server error, bo to raczej będzie wyglądało na błąd z naszej strony, aniżeli kogokolwiek innego (wpis jest w bazie, więc powinien mieć id - nie ma, znaczy że nasz błąd).

Reply:
No np o to mi chodziło

Reply:
okok, w takim razie jest, to zapraszam do rereview XD @Rados13